geonode/qgis_server/views.py

```diff
@@ -79,7 +79,7 @@ def download_zip(request, layername):
     return resp
 
 
-def legend(request, layername):
+def legend(request, layername, layertitle=None):
```

Comment:
Thanks, it's better like this.
How do we use that parameter ? This parameter is not the `urls.py`.